### ISSUE TITLE
Weird font issue on the panel

### ISSUE THREAD

**Issue by joaocandre** [lassekongo83/zuki-themes#155]
### Describe the issue

It seems to be some kind of weird font issue on the shell font face, which is visible on the colon (":") on the clock. Am I missing the correct font?
 
### Screenshot(s)

![](https://ibb.co/pfygb0Q)

### Versions

- OS/version: Archlinux
- GTK+  3.34.1

**Comment by lassekongo83** [comment]
`ttf-robot` is installed? Hinting set to Slight and Antialiasing set to Subpixel in gnome-tweaks?

**Comment by joaocandre** [comment]
Yes, all of those are installed and set.

**Comment by lassekongo83** [comment]
Are you on a HiDPI display, use a different font scaling than 1.0? (I'm unable to reproduce if I change mine). Do you have any old .fonts.conf file in your user directory?

**Comment by joaocandre** [comment]
@lassekongo83 No, this issue is happening on my laptop (pretty standard 1366x768 13.33' LCD display). I have noticed, however, that it is not present on my desktop displays (also not HiDPI). 

I have both `ttf-roboto` and `ttf-google-fonts`, could there be any sort of conflict or duplicate of the fonts used by the theme?

**Comment by lassekongo83** [comment]
`ttf-google-fonts-git` from AUR? It doesn't provide its own version of `ttf-roboto`, it's just listed as a dependency, so there shouldn't be any conflict unless the package messes with the fontconfig somehow, which I can't see.

**Comment by joaocandre** [comment]
@lassekongo83 Then I don't understand why this happens (I also don't have `ttf-robot-fontconfig` installed)  .
